test/__snapshots__/DatePicker.test.tsx.snap

```diff
@@ -27,16 +27,18 @@ exports[`DatePicker renders without errors 1`] = `
             </span>
           </label>
           <div
-            class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd css-154xyx0-MuiInputBase-root-MuiOutlinedInput-root"
+            class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-adornedEnd css-o9k5xi-MuiInputBase-root-MuiOutlinedInput-root"
           >
             <input
               aria-invalid="false"
+              autocomplete="off"
               class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
               id=":r0:"
+              inputmode="text"
               name="date"
-              placeholder="yyyy-mm-dd"
+              placeholder="YYYY-MM-DD"
               required=""
-              type="tel"
+              type="text"
```

Comment:
This looks correct, no idea how it got set to tel before!

Reply:
was thinking the same thing! 😄